### DISCUSSION TITLE
HSRC not getting connected to Wifi

### DISCUSSION THREAD

**Discussion by siddharthbhurat4** [HumanSupportRobot/QandA#40] [General]
I am trying to connect HSRC to a Wifi network. To connect I follow the following steps. First I put the robot in configuration mode and the required ip appears in ifconfig as per the documentation. But when I go to 192.168.0.215 to select a network and connect to, it never lists the network I want to connect to. (Not even the nearest mobile/laptop hotspot)
Then I tried going for the detailed configuration wherein I define ssid and password and then submit. But even this method doesnt work. 

Please can I know how to connect HSRC to required wifi or any other setting I can change to get it connected to wifi

**Comment by hsr-admin** [comment]
Not sure if this is helpful.
I have heard that in the past, some locations were unable to connect due to restrictions in the advanced settings of the Wireless Bridge installed in the HSR.

**Reply by ayuguchi** [reply]
Do you have any updates? I face the same problem. I would like to know the detail about the restrictions in the advanced settings of the Wireless Bridge.

**Reply by ayuguchi** [reply]
My problem has been solved. This problem occurred because my understanding of how to connect to WiFi was not enough.
